themes/10up-block-theme/assets/css/base/reset.css

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
I've been adding this to my resets as well to ensure alt tags remain readable if the image doesn't load.  Sometimes you see folks put `font-size: 0;` or `line-height: 0` on figure/div tags to avoid that small bit of white space that can occur, this protects against that situation. 

```css
img {
  font-size: 1rem;
  font-style: italic;
  line-height: 1.5;
}
```